README.md

```diff
@@ -61,7 +61,7 @@ maps to CMAKE_BUILD_TYPE, and supports the following modes
 
 Note that seastar is more sensitive to allocators and optimizations than
 usual. A quick rule of the thumb of the relative performances is that
-release is 2 times faster than dev, 150 times faster than sanitize and
+release is 2 times faster than dev, 150 times faster than sanitizing and,
```

Comment:
Again, a wrong correction. "sanitize" is the name of a build mode, just like the "dev" and "release" that precede it. It's not a verb that you can inflect...
It probably makes sense to quote these build mode names - "release", "dev" and "sanitize". That would be a better fix. This sentence can also be improved by using the phrase "build mode" somewhere, not just words like "release" et al.
If you're changing this paragraph, also the word "performances" shouldn't be plural :-(